ingress-watcher/README.md

```diff
@@ -0,0 +1,54 @@
+ingress-watcher
+===============
+
+`ingress-watcher` is a watching agent for L7 endpoints.
+
+Usage
+-----
+
+### Start `ingress-watcher`
+
+`ingress-watcher` exports metrics in one of the following two ways:
+
+1. Run a metrics server and return metrics at `GET /metrics`.
+    ```bash
+    ingress-watcher export \
+    --target-urls https://example.com \
+    --target-urls https://example.org \
+    --listen-addr localhost:8080 \
+    --watch-interval 10s
+    ```
+
+2. Push and expose the collected metrics via [Pushgateway](https://github.com/prometheus/pushgateway).
+    ```bash
+    ingress-watcher push \
+    --target-urls https://example.com \
+    --target-urls https://example.org \
+    --push-addr localhost:9091 \
+    --watch-interval 10s \
+    --job-name job \
+    --push-interval 5s
```

Comment:
Why can you set the `--watch-interval` and `--push-interval` to different values?
It will cause a duplicate value push.
I think pushing should be at the same time as watching.

Reply:
`--watch-interval` indicates a sampling rate of checking Ingress. On the other hand, `--push-interval` means updating interval of metrics' counter.

So, for example, a setting such as smaller `--watch-interval` and larger `--push-interval` make sense (the example in `README.md` is not good, so I'll rewrite it).